hotline/templates/notifications/list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+{% load notifications %}
+{% block content %}
+<div class="row">
```

Comment:
Any particular reason you're using all that markup? `<div class='row'>` is useful when you have columns (i.e. you want things side-by-side). But if the element spans the whole container, you don't need it.